src/Microsoft.ApplicationInsights.AspNet/SdkVersionUtils.cs

```diff
@@ -1,17 +1,33 @@
 ﻿namespace Microsoft.ApplicationInsights.AspNet
-{ 
+{
     using System.Linq;
     using System.Reflection;
 
     internal class SdkVersionUtils
     {
-        internal const string VersionPrefix = "aspnet5: ";
+        internal const string VersionPrefix = "aspnet5-";
+
+        internal static string GetFrameworkType()
+        {
+            string framework;
+#if dnx451
+           // F representing the full framework
+           framework = "F:"; 
+#elif dnxcore50
+            // C representing the core framework
+            framework = "C:";
+#else
+          // O representing other framework
+          framework = "O:";
+#endif
```

Comment:
The reason I kept there for now, is because we might have netstandard1.3, which can support "dotnet5.4" application for example, or "netstandardapp1.5". I am not sure how is it going to take into account all those versions. We can remove that once we are sure about this details.